compiler-rt/lib/interception/interception_win.cpp

```diff
@@ -829,6 +832,8 @@ static size_t GetInstructionSize(uptr address, size_t* rel_offset = nullptr) {
     case 0x588948:    // 48 89 58 XX : mov QWORD PTR[rax + XX], rbx
     case 0xec8348:    // 48 83 ec XX : sub rsp, XX
     case 0xf88349:    // 49 83 f8 XX : cmp r8, XX
+    case 0x148d4e:    // 4e 8d 14 XX : lea r10, [rcx+r8*XX]
```

Comment:
nit: That last byte doesn't just control the scale (`*XX`) but the base register (`rcx`) and index register (`r8`) as well. Maybe we should write it as `lea r10, [...]`

Reply:
Thanks, I will change it. Now when I look at it, I guess I should change `49 8d 48 ff` and `4c 8d 04 11` into a generic version too?

Reply:
Yes that sounds like a good idea.